phase02-code review/src/main/java/HashInvertedIndex.java

```diff
@@ -0,0 +1,54 @@
+import opennlp.tools.stemmer.PorterStemmer;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class HashInvertedIndex implements InvertedIndex{
+    private static HashInvertedIndex instance;
+    private final HashMap<String, ArrayList<String>> words;
+    private final PorterStemmer porterStemmer;
+    private final FileReader fileReader;
+
+    public HashInvertedIndex() {
+        words = new HashMap<>();
+        porterStemmer = new PorterStemmer();
+        fileReader = FileReader.getInstance();
+        organize();
+    }
+
+    public void organize() {
+        HashMap<String, String> docs = fileReader.getDocs();
+        for (final String doc : docs.keySet()) {
+            query(doc, docs.get(doc));
+        }
+    }
+
+    private void query(final String docName, final String content) {
```

Comment:
leave a explanation for this function here

Reply:
I think if I rename the function it would be better.